pkg/api/v1beta3/types.go

```diff
@@ -995,6 +1006,16 @@ type Binding struct {
 	Target ObjectReference `json:"target" description:"an object to bind to"`
 }
 
+// DeleteOptions may be provided when deleting an API object
+type DeleteOptions struct {
+	TypeMeta `json:",inline"`
```

Comment:
I know it has to be versioned, but I imagine we'll typically just infer the version from the URL path. 

I'm fine with ObjectMeta, however.